Dockerfile

```diff
@@ -1,22 +1,44 @@
-FROM python:2.7-slim
+FROM alpine:3.3
```

Comment:
:+1: nice! I generally go with debian:stable these days instead of python b/c I usually need packages not available in alpine. But in the case it's a perfect match.